app/views/admin/attachments/edit.html.erb

```diff
@@ -0,0 +1,16 @@
+<% if @attachment.errors.present? %>
+  <% @attachment.errors.full_messages.each do |message| %>
+    <%= message %>
+  <% end %>
+<% end %>
+
+<div class="block-form">
+  <%= form_for @attachment, url: { controller: "admin/attachments", action: "update" } do |f| %>
+    <%= f.label :name, 'Name:' %>
+    <%= f.text_field :name %>
+    <br />
```

Comment:
Not critical, but better to use CSS styling  to make empty space between elements instead of **br** tag.

Reply:
fixed